charts/shield/templates/host/_configmap_helpers.tpl

```diff
@@ -100,7 +100,9 @@ true
 {{- $config := dict
   "k8s_cluster_name" .Values.cluster_config.name
   "collector" (include "common.collector_endpoint" .)
-  "collector_port" .Values.sysdig_endpoint.collector.port }}
+  "collector_port" .Values.sysdig_endpoint.collector.port
+  "k8s_delegated_nodes" (dig "k8s_delegated_nodes" 0 .Values.host.additional_settings)
```

Comment:
I imagine we'd want to have a check to see if `features.kubernetes_metadata.enabled` has been set to `false` here. If this feature is turned off I think we'd like the agent/host shield to run with the default values (unless manually overridden of course).

Reply:
@mavimo @irozzo-1A WDYT?

Reply:
Yep, I agree that would be better to use the default if `kubernetes_metadata` is disabled

Reply:
Ok, i did apply the changes. User specified configuration will alway rule but, as a default, if the feature is enabled we do force the delegated agents to 0 and, if not, we simply don't set that configuration.